tests/test_schema.py

```diff
@@ -485,6 +485,16 @@ def test_leaf_type_fraction_digits(self):
         self.assertIsInstance(t, Type)
         self.assertEqual(next(t.all_fraction_digits(), None), 2)
 
+    def test_leaf_type_require_instance(self):
+        self.ctx.destroy()
+        self.ctx = Context(YANG_DIR, leafref_extended=True)
+        self.ctx.load_module("yolo-leafref-extended")
+        leaf = next(self.ctx.find_path("/yolo-leafref-extended:ref3"))
```

Comment:
Do you really need leafref_extended to have require instance? If not, can you add this test case in another file, and have it test only the require_instance part?